paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentMethodIcon.kt

```diff
@@ -51,24 +59,14 @@ internal fun PaymentMethodIcon(
                 imageLoader = imageLoader,
                 contentDescription = null,
                 contentScale = ContentScale.Fit,
-                errorContent = {
-                    PaymentMethodIconFromResource(
-                        iconRes = iconRes,
-                        colorFilter = colorFilter,
-                        modifier = iconModifier,
-                        alignment = contentAlignment,
-                    )
-                },
+                loadingContent = { iconFromResource() },
+                errorContent = { iconFromResource() },
+                disableAnimations = true,
```

Comment:
if we don't disable animations here, then the icon will flicker as it animates from the loading content to the loaded content (even though they are the same)